### PR TITLE
Beit postprocessing

### PR DESCRIPTION
**Pull request by alaradirik**
# What does this PR do?
Adds a post-processing method to BeiTFeatureExtractor for semantic segmentation.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [X ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [X ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: src/transformers/models/beit/feature_extraction_beit.py [hunk 1]
```diff
@@ -222,3 +225,44 @@ def __call__(
         encoded_inputs = BatchFeature(data=data, tensor_type=return_tensors)
 
         return encoded_inputs
+
+    def post_process_semantic_segmentation(self, outputs, target_sizes: Union[TensorType, List[Tuple]] = None):
+        """
+        Parameters:
+        Converts the output of [`BeitForSemanticSegmentation`] into semantic segmentation maps. Only supports PyTorch.
```
Comment (sgugger) [inline src/transformers/models/beit/feature_extraction_beit.py:232]: Should be before `Parameters`

File: src/transformers/models/beit/feature_extraction_beit.py [hunk 2]
```diff
@@ -222,3 +225,44 @@ def __call__(
...
+            outputs ([`BeitForSemanticSegmentation`]):
+                Raw outputs of the model.
+            target_sizes (`torch.Tensor` of shape `(batch_size, 2)` or `List[Tuple]` of length `batch_size`,
+            *optional*):
```
Comment (sgugger) [inline src/transformers/models/beit/feature_extraction_beit.py:236]:
```suggestion
            target_sizes (`torch.Tensor` of shape `(batch_size, 2)` or `List[Tuple]` of length `batch_size`, 0*optional*):
```
Needs to be on the same line!

**Review by sgugger — APPROVED** [review]
LGTM, just one docstring is badly formatted.

File: src/transformers/models/beit/feature_extraction_beit.py (hunk 2 above)
Comment (alaradirik) [reply src/transformers/models/beit/feature_extraction_beit.py]: Thanks, fixed it!

**Pull request merged**